### PR TITLE
Add statping guide, update cryptpad

### PR DESCRIPTION
**Pull request by GerJuli**

### CODE REVIEW

File: source/guide_statping.rst [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+::
+
+  [isabell@stardust ~]$ mkdir ~/statping
+
+
+Create a database
+
+::
+
+  [isabell@stardust ~]$ mysql -e "CREATE DATABASE eliza_statping"
```
Comment (luto) [inline source/guide_statping.rst]: Please use `${USER}_statping`, so the command can be copy-pasted.

File: source/guide_statping.rst [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  * :manual:`supervisord <daemons-supervisord>`
+
+Prerequisites
+=============
+
+Set up the backends:
+
+::
+
+  [isabell@stardust ~]$ uberspace web backend set / --http --port 8080
```
Comment (luto) [inline source/guide_statping.rst]: We've got a snippet for web back backends. You can use it like so:

```
Configure web server
--------------------

.. note::

    Django is running on port 8000.

.. include:: includes/web-backend.rst
```

File: source/guide_statping.rst [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+   Configuration site of statping.
+
+Select the Database Connection :manual:`MYSQL <database-mysql>`  and type in your your username and password.
+The database runs on localhost, the port is 3306.
+
+If you don't remember your password use
+
+.. code-block:: console
+
+   [eliza@dolittle ~]$ my_print_defaults client
```
Comment (luto) [inline source/guide_statping.rst]: There is a snippet for mysql, which is usually included [near the top](https://lab.uberspace.de/guide_etherpad.html#prerequisites). Do you mind using that one?

```
.. include:: includes/my-print-defaults.rst
```

**Comment by luto** [comment]
The comments are marked as resolved, but the guide didn't actually change (the backend call is still in there, so is `my_print_defaults`). Did you push to the right branch?

**Comment by GerJuli** [comment]
Sorry, the commits added today should resolve the issues. Thanks for the feedback!

**Comment by luto** [comment]
Thank you! :tada:

**Pull request closed without merging**